.env.sample

```diff
@@ -1,2 +1,6 @@
-DB_URL=postgresql://postgres:postgres@localhost:5432/postgres
 DUNE_API_KEY=
+DB_URL=postgresql://postgres:postgres@localhost:5432/postgres
+
+QUERY_ID=
+POLL_FREQUENCY=10
+QUERY_ENGINE=medium
```

Comment:
TODO: Query engine will be an optional configuration parameter which defaults to medium. There are also only two possible values here (medium and large). Note that only paid plans have access to the large performance option.

Reply:
Also related to #4 (when coming up with the configuration structure).